cloud/pkg/edgecontroller/controller/downstream.go

```diff
@@ -120,7 +120,6 @@ func (dc *DownstreamController) syncConfigMap(stop chan struct{}) {
 				default:
 					// unsupported operation, no need to send to any node
 					klog.Warningf("config map event type: %s unsupported", e.Type)
-					break
```

Comment:
if it is unsupported operation,  should we not continue to send messages?

Reply:
if should not send message,  we can use `continue ` + `label`

Reply:
Good point! I re-checked and think it should break the outer for loop, directly go to waiting next event.
Modified. Thanks.